test/evergreen/compatibility_test_for_releases.sh

```diff
@@ -676,6 +690,134 @@ top="test-compatibility-run"
 rm -rf "$top" && mkdir "$top"
 cd "$top"
 
+#############################################################
+# create_file:
+#       arg1: branch
+#       arg2: file
+#############################################################
+create_file()
+{
+    echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+    echo "Branch \"$1\" creating and populating \"$2\""
+    echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+
+    wt_cmd="$1/build/wt"
+    test_dir="$1/build/WT_TEST/"
+    uri="file:$2"
+
+    # Make the home directory.
+    mkdir -p $test_dir
+
+    # Create the file and populate with a few key/values.
+    $wt_cmd -h $test_dir create -c "key_format=S,value_format=S" $uri
+    $wt_cmd -h $test_dir write $uri abc 123 def 456 hij 789
+}
+
+#############################################################
+# import_file:
+#       arg1: dest branch
+#       arg2: source branch
+#       arg3: file
+#############################################################
+import_file()
+{
+    echo "=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-=-="
+    echo "Importing file \"$3\" from \"$1\" to \"$2\""
```

Comment:
Should it be "Importing file \"$3\" from \"$2\" to \"$1\"" instead? It feels more intuitive to have arg1 being the source and arg2 the destination :)

Reply:
Good catch, thanks for that. Yeah I agree with your sentiment about that, I'll make changes to other functions if needed as well.